client/src/views/users/UserDetailsModal.vue

```diff
@@ -0,0 +1,206 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page class="modal">
+    <ms-modal
+      :title="$t('UsersPage.UserDetailsModal.title')"
+      :close-button="{ visible: true }"
+    >
+      <div>
+        <ion-text class="text1">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.name') }}
+        </ion-text>
+        <ion-text class="text1-right">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.joined') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-text class="text2">
+          {{ user.humanHandle.label }}
+        </ion-text>
+        <ion-text class="text2-right">
+          {{ timeSince(user.createdOn, '--', 'short') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-text class="text4">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.commonWorkspaces') }}
+        </ion-text>
+      </div>
```

Comment:
Here the BEM suggestion.
In addition, it is preferable to group the code according to context so that each group/block can be managed (such as the user's name and title).

```suggestion
      <div class="detail-name">
        <ion-text class="details-name__title">
          {{ $t('UsersPage.UserDetailsModal.subtitles.name') }}
        </ion-text>
        <ion-text class="detail-name__fullname">
          {{ user.humanHandle.label }}
        </ion-text>
      </div>
      <div class="detail-joined">
        <ion-text class="detail-joined__title">
          {{ $t('UsersPage.UserDetailsModal.subtitles.joined') }}
          {{ user.humanHandle.label }}
        </ion-text>
        <ion-text class=class="detail-joined__date">
          {{ timeSince(user.createdOn, '--', 'short') }}
        </ion-text>
      </div>
      <div>
        <ion-text class="text4">
          {{ $t('UsersPage.UserDetailsModal.subtitles.commonWorkspaces') }}
        </ion-text>
      </div>
```